src/bijectors/radial_layer.jl

```diff
@@ -64,8 +68,8 @@ end
 
 function (ib::Inversed{<:RadialLayer})(y)
     flow = ib.orig
-    α = softplus(flow.α_[1])            # from A.2
-    β_hat = - α + softplus(flow.β[1])   # from A.2
+    α = softplus.(flow.α_[1])            # from A.2
+    β_hat = - α + softplus.(flow.β[1])   # from A.2
```

Comment:
No need for broadcasting?

Reply:
Both are 1x1 matrices, so no. Should we fuse anyways so that the entire computation is fused, incl softplus?

Reply:
Scalar operations don't benefit from fusing.

Reply:
Ah, true :+1: